README.md

```diff
@@ -147,6 +147,26 @@ And that's it!
 
 If you're using more than one service at a time in your code and get two different `ibm-credentials.env` files, just put the contents together in one `ibm-credentials.env` file and the SDK will handle assigning credentials to their appropriate services.
 
+**Special Note**: Due to legacy issues in Assistant V1 and V2 as well as Visual Recognition V3 and V4, the following parameter `serviceName` must be added when creating the service object:
+```js
+const AssistantV1 = require('ibm-watson/assistant/v1');
+const assistant = new AssistantV1({
+  version: '2020-04-01',
+  serviceName: 'assistant',
+})
+```
+```js
+const VisualRecognitionV3 = require('ibm-watson/visual-recognition/v3');
+const assistant = new VisualRecognitionV3({
+  version: '2018-03-19',
+  serviceName: 'visual-recognition',
+})
+```
+It is worth noting that implementing this will prioritize reading the credentials file over using the `VCAP_SERVICES` variable when pushing to IBM Cloud. This will become an issue if you do not push a credentials file to the cloud as VCAP does not know how to interpret these `serviceName` parameters. Credential priority follows this order: 
```

Comment:
I think there was a slight miscommunication here. The `serviceName` parameter does not impact priority at all so it won't specifically prioritize the credentials file. The VCAP reader works fine with the service name. The problem is, the VCAP object will never have keys for "assistant" or "visual-recognition". So, if the user intends to use the VCAP object on the cloud, they should not specify the service name